runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkPipelineRunner.java

```diff
@@ -170,13 +169,16 @@ public static void main(String[] args) throws Exception {
     Pipeline pipeline = PortablePipelineJarUtils.getPipelineFromClasspath(baseJobName);
     Struct originalOptions = PortablePipelineJarUtils.getPipelineOptionsFromClasspath(baseJobName);
 
-    // Flink pipeline jars distribute and retrieve artifacts via the classpath.
-    PortablePipelineOptions portablePipelineOptions =
-        PipelineOptionsTranslation.fromProto(originalOptions).as(PortablePipelineOptions.class);
-    portablePipelineOptions.setRetrievalServiceType(RetrievalServiceType.CLASSLOADER);
-    String retrievalToken = PortablePipelineJarUtils.getArtifactManifestUri(baseJobName);
+    // The retrieval token is only required by the legacy artifact service, which the Flink runner
+    // no longer uses.
+    String retrievalToken =
+        ArtifactApi.CommitManifestResponse.Constants.NO_ARTIFACTS_STAGED_TOKEN
```

Comment:
Nit: Should we pull this out as a constant somewhere (as we do with the other URNs)?

Reply:
I won't since this part of the API is deprecated.